src/gui/DatabaseWidget.cpp

```diff
@@ -530,6 +548,19 @@ void DatabaseWidget::deleteEntries()
     }
 }
 
+QString DatabaseWidget::referencesMessage(QMap<Entry*, int> const& referencesMap)
```

Comment:
This function is unnecessary if you follow my implementation suggestion above.

Reply:
Agreed.